api/pkg/service/catalog/syncer.go

```diff
@@ -255,6 +255,7 @@ func (s *syncer) updateResources(
 
 		log.Infof("Resource: %s  ID: %d stored", r.Name, dbRes.ID)
 
+		s.updateResourceCategory(txn, log, &dbRes, r.Categories)
```

Comment:
can we pass resource id here instead of `dbRes`

Reply:
I think in order to maintain consistency wrt to other methods passed the resource object
Shall I still pass the id ?

Reply:
no, okay then use the resource object